verification/verify_test.go

```diff
@@ -300,9 +300,9 @@ func TestVerifyPixels(t *testing.T) {
 
 	// Test no writing temp file with incorrect pixels
 	err = verifyPixels("../server/test.flv", nil, 50)
-	assert.Equal(ErrPixelMismatch, err)
+	assert.EqualError(err, "Invalid data found when processing input")
 
 	// Test no writing temp file with correct pixels
 	err = verifyPixels("../server/test.flv", nil, p)
-	assert.Nil(err)
+	assert.EqualError(err, "Invalid data found when processing input")
```

Comment:
Since `countPixels()` will return this error whenever nil data is passed and since the above test already handles this case, I don't think we need this test anymore.

Reply:
done in 98113590b8478473e3b10cf74693555dba91edb0